### ISSUE TITLE
Plugin makes iabbreviations stop working

### ISSUE THREAD

**Issue by eater** [dkarter/bullets.vim#7]
When I have bullets.vim loaded, the iabbreviations I have defined don't execute on `<CR>`, because it's imapped.

**Comment by eater** [comment]
A fix for this would be to add a `<C-]>` trigger on line 379:

```
call s:add_local_mapping('inoremap', '<cr>', '<C-]><C-R>=<SID>insert_new_bullet()<cr>')
```

**Comment by dkarter** [comment]
Thanks for bringing this to my attention. I usually expand iabbrevs with space so was not aware this breaks abbreviations, but it should be fixed. Im not sure I understand what '<C-]><C-R>=' does. But I'll test if it works and lookup the documentation. Also PRs are welcome :)

**Comment by dkarter** [comment]
Fixed with #9
